frontend/src/app/components/ExperimentRowCard.js

```diff
@@ -81,14 +81,14 @@ export default class ExperimentRowCard extends React.Component {
     if ((now - modified) > MAX_JUST_UPDATED_PERIOD) { return false; }
 
     // If modified since the last time seen, *do* consider it updated.
-    if (modified > experiment.lastSeen) { return true; }
+    if (modified > getExperimentLastSeen(experiment)) { return true; }
```

Comment:
We lost this from ye olde Ampersand site, [over here](https://github.com/mozilla/testpilot/blob/2016-09-12/frontend/src/app/models/experiment.js#L44). Just realized it wasn't working as I wrote the test.